docs/spec.md

```diff
@@ -1979,7 +1992,8 @@ If `feature_group_count = 1` and `batch_group_count = 1`, then for all
 `output_spatial_index` in the index space of `dim(result, output_spatial_dimensions)`,
 `result[result_shape(:, output_spatial_index, :)] = dot_product` where:
 
-* `padded_lhs = pad(lhs, 0, lhs_padding[:, 0], lhs_padding[:, 1], lhs_base_dilations[:] - 1)`.
+* `padded_lhs_value = is_quantized_tensor(lhs) ? zero_point(lhs) : 0`.
```

Comment:
How about renaming to `padding_value`, instead of `padded_value`, to match the name of the corresponding input of PadOp?